src/clr-core/button/button.stories.ts

```diff
@@ -0,0 +1,235 @@
+/*
+ * Copyright (c) 2016-2020 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import '@clr/core/button';
+import { ClrLoadingState } from '@clr/core/button';
+import { cssGroup, propertiesGroup, setStyles } from '@clr/core/common';
+import '@clr/core/icon';
+import { ClarityIcons, userIcon } from '@clr/core/icon-shapes';
+import { action } from '@storybook/addon-actions';
+import { boolean, color, select, text } from '@storybook/addon-knobs';
+import { html } from 'lit-html';
+
+ClarityIcons.addIcons(userIcon);
+
+export default {
+  title: 'Components|Button',
+  component: 'cwc-button',
+  parameters: {
+    options: { showPanel: true },
+    design: {
+      type: 'figma',
+      url: 'https://www.figma.com/file/v2mkhzKQdhECXOx8BElgdA/Clarity-UI-Library---light-2.2.0?node-id=20%3A0',
```

Comment:
Storybook we can link the related figma file and embed it in the API doc example